array-manipulation-3/src/equal.js

```diff
@@ -0,0 +1,25 @@
+/* exported equal */
+
+// create a counter to track the first array position in the first array
+// loop through both arrays
+// if each one does not match when the loop counter is in the same place, return false
+// if they do match, increment the counter for the first array and continue the loop
+// if the end of the array is reached, return true
+// if the 2 arrays are not the same length, return false
+// if the arrays are empty, return true
+
+function equal(first, second) {
+  let f = 0;
+  for (let s = 0; s < second.length; s++) {
+    if (f === first.length - 1 && first[first.length === second[second.length]]) {
```

Comment:
I would do the check to see if the arrays are the same length before the loop, so you only need to check once, not on every iteration.

Reply:
part of this is that is also has to check the values at the end of the array or it doesn't work at all. it does 2 things.